 Customize live image for oe:

 * disable ssl verification
 * customize repo path

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Rebase for anaconda 39 on 20240304.

Signed-off-by: Kai Kang <kai.kang@windriver.com>

---
 pyanaconda/core/constants.py                       | 2 +-
 pyanaconda/modules/common/structures/live_image.py | 2 +-
 pyanaconda/modules/payloads/source/cdrom/cdrom.py  | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/pyanaconda/core/constants.py b/pyanaconda/core/constants.py
index 3b7756698f..873e840cf3 100644
--- a/pyanaconda/core/constants.py
+++ b/pyanaconda/core/constants.py
@@ -49,7 +49,7 @@ MOUNT_DIR = "/media/realroot"
 DRACUT_ISODIR = "/run/install/source"
 DRACUT_REPO_DIR = "/run/install/repo"
 ISO_DIR = MOUNT_DIR + "/isodir"
-SOURCES_DIR = MOUNT_DIR + "/sources"
+SOURCES_DIR = "/run/install/sources"
 
 # Names of repositories.
 BASE_REPO_NAME = "anaconda"
diff --git a/pyanaconda/modules/common/structures/live_image.py b/pyanaconda/modules/common/structures/live_image.py
index b47e6252c5..6b3d37862e 100644
--- a/pyanaconda/modules/common/structures/live_image.py
+++ b/pyanaconda/modules/common/structures/live_image.py
@@ -30,7 +30,7 @@ class LiveImageConfigurationData(DBusData):
         self._url = ""
         self._proxy = ""
         self._checksum = ""
-        self._ssl_verification_enabled = True
+        self._ssl_verification_enabled = False
 
     @property
     def url(self) -> Str:
diff --git a/pyanaconda/modules/payloads/source/cdrom/cdrom.py b/pyanaconda/modules/payloads/source/cdrom/cdrom.py
index f1def79791..dd6c3fa13c 100644
--- a/pyanaconda/modules/payloads/source/cdrom/cdrom.py
+++ b/pyanaconda/modules/payloads/source/cdrom/cdrom.py
@@ -93,7 +93,7 @@ class CdromSourceModule(PayloadSourceBase, MountingSourceMixin, RPMSourceMixin):
 
     def generate_repo_configuration(self):
         """Generate RepoConfigurationData structure."""
-        return RepoConfigurationData.from_directory(self.mount_point)
+        return RepoConfigurationData.from_directory(self.mount_point + '/Packages')
 
     def set_up_with_tasks(self):
         """Set up the installation source.
